pdr_backend/lake/fetch_ohlcv.py

```diff
@@ -55,6 +60,45 @@ def safe_fetch_ohlcv_ccxt(
         return None
 
 
+@enforce_types
+def safe_fetch_ohlcv_dydx(
+    symbol: str,
+    resolution: str,
+    st_ut: UnixTimeMs,
```

Comment:
Key comment:

This method should have exactly the same interface / args as safe_fetch_ohlcv(). In particular, a "since" and "limit", and no "fin_ut".

This also implies that this routine has much less work to do. Eg it doesn't need to loop over and over to gather data. That's done by the data factory itself.